cmd/utils.go

```diff
@@ -160,6 +161,24 @@ var globalProfiler interface {
 	Stop()
 }
 
+// Prints a given message after a requested delay.
+func delayPrint(message string, delay time.Duration) chan bool {
```

Comment:
I am okay with that. But I want to get a conclusion from @krishnasrinivas . So current PR is basically nothing but what @krishnasrinivas proposed.